crates/aptos/Cargo.toml

```diff
@@ -97,3 +97,6 @@ cli-framework-test-move = []
 
 [build-dependencies]
 shadow-rs = { workspace = true }
+
+[lib]
+crate-type = ["rlib", "dylib"]
```

Comment:
This is being used to generate the Rust shared library, which can be consume by the SDK (i.g Typescript)